src/c++/perf_analyzer/genai-perf/tests/test_llm_inputs.py

```diff
@@ -248,6 +248,7 @@ def test_create_openai_to_completions(self):
 
         assert pa_json is not None
         assert len(pa_json["data"]) == LlmInputs.DEFAULT_LENGTH
+        assert isinstance(pa_json["data"][0]["payload"][0]["prompt"], str)
```

Comment:
For future reference, can we add a short comment on why we only support string?

Reply:
added. good call out